cekit/builders/docker_builder.py

```diff
@@ -152,38 +142,51 @@ def _build_with_docker(self, docker_client):
             if "To enable Red Hat Subscription Management repositories:" in ' '.join(build_log) and \
                     not os.path.exists(os.path.join(self.target, 'image', 'repos')):
                 msg = "Image build failed with a yum error and you don't " \
-                      "have any yum repository configured, please check " \
+                      "have any yum repository qured, please check " \
```

Comment:
Spelling?

Reply:
Yah, files besides `cli.py` were not entirely scanned yet for issues with spelling tools and pylint.